zerver/openapi/zulip.yaml

```diff
@@ -11575,6 +11596,10 @@ paths:
                           Present if `realm_user` is present in `fetch_event_types`.
 
                           The full name of the current user.
+                      send_private_message_on_invitee_signup:
+                        type: boolean
+                        description: |
+                          Whether to get a PM when an invitee join's the realm.
```

Comment:
It should be "joins" not "join's". This should be changed in other places as well.

Reply:
You should also add the correct (as described in above comment) `Changes` entry.